bazel/emscripten_deps.bzl

```diff
@@ -40,7 +40,7 @@ filegroup(
         "bin/wasm-emscripten-finalize{bin_extension}",
         "bin/wasm-ld{bin_extension}",
         "bin/wasm-opt{bin_extension}",
-   ],
+   ] + glob(["emscripten/node_modules/**"]),
```

Comment:
BTW, I still don't understand how the linker can work without the libraries in `emscripten/cache/sysroot/lib`